content/Live Streaming Apps Architecture - Part 1 - German Lopez.md

```diff
@@ -0,0 +1,182 @@
+<img src="images/live-streaming-architecture.png" alt="Live Streaming Architecturing"/>
+
+# Live Streaming apps architecture - Part 1
+
+**Abstract:** A few ideas and considerations for architecting a live streaming application based on our previous experience.
+
+## Live streaming
+
+The term implies that the media(audio, video or both) is broadcasted to the end users in real time. This is different than just _streaming_ which could refer to previously recorded content such as VOD(Video On Demand), music streaming, etc. The later means that the media is delivered and played on demand rather than being generated simoultaneously.
+
+Is this difference what makes **live streaming** a lot harder to design, implement and scale.
+
+## From our experience
+
+There are different approaches you can take to build your live streaming platform:
+
+1. Implement your own.
+2. Use one of the existing [**Platform as a Service (PaaS)**](#paas) solutions.
+3. Use an existing software solution that you can run on your own **Infrastructure as a Service (IaaS)**.
+
+While the first option sounds complicated (it is), it might be the best choice if you have the time, resources, and super specific requirements for your live streaming app. Since I wouldn't recommend to code the entire streaming software from scratch; a good starting point would be the [open source solutions](#open-source).
```

Comment:
```suggestion
While the first option sounds complicated (it is), it might be the best choice if you have the time, resources, and super specific requirements for your live streaming app. Since I wouldn't recommend coding the entire streaming software from scratch; a good starting point would be the [open source solutions](#open-source).
```